roles/openshift_prometheus/README.md

```diff
@@ -23,6 +23,10 @@ For default values, see [`defaults/main.yaml`](defaults/main.yaml).
 
 - `openshift_prometheus_<COMPONENT>_image_version`: specify image version for the component 
 
+- `openshift_prometheus_hostname`: specify the hostname for the route to prometheus `prometheus-{{openshift_prometheus_namespace}}.{{openshift_master_default_subdomain}}`
+
+- `openshift_prometheus_alerts_hostname`: specify the hostname for the route to prometheus `prometheus_alerts-{{openshift_prometheus_namespace}}.{{openshift_master_default_subdomain}}`
```

Comment:
'specify the hostname for the route to prometheus-alerts'

Reply:
true... copy-pasted too fast...